src/Map/main.vala

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1 --pkg shumate-1.0
+
+private Shumate.SimpleMap map_widget;
+private Shumate.Viewport viewport;
+private Gtk.Entry entry_latitude;
+private Gtk.Entry entry_longitude;
+
+void go_to_location () {
+    var latitude = float.parse (entry_latitude.text);
+    var longitude = float.parse (entry_longitude.text);
+
+    if (!float.try_parse (entry_latitude.text) || !float.try_parse (entry_longitude.text)) {
+        message (@"Please enter valid coordinates");
+        return;
+    }
```

Comment:
Use the out parameter of the `try_parse` method so the parsing is performed if possible.
```suggestion
    float latitude = 0, longitude = 0;

    if (!float.try_parse (entry_latitude.text, out latitude) || !float.try_parse (entry_longitude.text, out longitude)) {
        message (@"Please enter valid coordinates");
        return;
    }
```